streams/src/main/java/org/apache/kafka/streams/processor/internals/assignment/AssignorConfiguration.java

```diff
@@ -261,4 +257,24 @@ public InternalTopicManager getInternalTopicManager() {
     public CopartitionedTopicsEnforcer getCopartitionedTopicsEnforcer() {
         return copartitionedTopicsEnforcer;
     }
+
+    public AssignmentConfigs getAssignmentConfigs() {
```

Comment:
req: Please add unit tests for this method (I am aware that there aren't any unit tests for this class).

Reply:
ack